server/db/seeders/20190612011456-borrowed-book.js

```diff
@@ -10,40 +10,43 @@ export function up(queryInterface) {
       {
         isbn: 44537,
         title: 'As a man thinketh',
-        patronId: 4,
+        patronId: 5,
```

Comment:
Why was this and other related changes required?

Reply:
These adjustments were made for the test purposes since some books had already been borrowed I needed a user that hadn't borrowed any books.
Secondly, this test runs before the user tests, so we only have 5 users before the test runs.